.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 88
+ignore = E203, W503
```

Comment:
For VS code, go to `code > preferences > settings` and search for `Insert Final Newline`

Reply:
oh I thought I had that on... maybe it was only local to a project and not the global config.

Reply:
done